manifests/t/Taozuhong/Kangaroo/1.98.1.220904/Taozuhong.Kangaroo.yaml

```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=AUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Taozuhong.Kangaroo
+PackageVersion: 1.98.1.220904
```

Comment:
```suggestion
PackageVersion: 1.98.1.220905
```
```
Enter a pull request number: https://github.com/microsoft/winget-pkgs/pull/78494

Manifest validation succeeded.
Found Kangaroo [Taozuhong.Kangaroo] Version 1.98.1.220904
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://github.com/dbkangaroo/kangaroo/releases/download/v1.98.1.220904/kangaroo-1.98.1.220904-AMD64.exe
  ██████████████████████████████  37.4 MB / 37.4 MB
Successfully verified installer hash
Starting package install...
Successfully installed

Name          : Kangaroo 1.98.1.220905
Publisher     : Taozuhong
Version       : 1.98.1.220905
ProductCode   : Kangaroo
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Kangaroo
Uninstall     : winget uninstall --id "Kangaroo"

Press any key to continue . . .
```

Reply:
Note: You'll also need to change `manifests/t/Taozuhong/Kangaroo/1.98.1.220904` to `manifests/t/Taozuhong/Kangaroo/1.98.1.220905`